scriptworker/constants.py

```diff
@@ -31,6 +31,16 @@
     'mitchhentges',
 )
 
+_COT_VERSION_DESCRIPTIONS = frozendict({
+    1: 'Initial Chain of Trust implementation with GPG signatures',
+    2: 'Json-e verification support',
+    3: 'Action hook support',
+    4: 'Release promotion action hook support',
+    5: 'Initial ed25519 support; GPG deprecated',
+    6: 'Drop support for non-hook actions',
+    7: 'Drop support for GPG',
+})
```

Comment:
We have `min_cot_version` which corresponds to the keys. The values are human documentation. I can remove this, but I'd like this documented somewhere. Do you think this works, or do you think it belongs elsewhere?

Reply:
4 just landed. This is 5. 6 and 7 aren't ready to land yet, and yes, they could switch order depending on when they land, but this is the current plan. I sometimes reference cot versions by number; this is the map from number to what happened in that version.

Reply:
It's easy enough to move this to the docs though.

Reply:
I was looking at a `chain-of-trust.json` file, and I noticed that that has a version number in it, which still appears to be v1, so that is perhaps a source of confusion around what CoT versioning is.

Reply:
The json schema is still backwards compatible with the first cot artifact schemas, hence v1. However, the logic behind how to verify the chain has changed drastically: from guessing at whether the task looks like it might have come from the tree, to using json-e to actually regenerate the task, to supporting action hooks, to supporting action hooks for relpro. Not all of these versions are backwards compatible.